packages/arb-token-bridge-ui/src/state/cctpState.ts

```diff
@@ -730,7 +730,7 @@ export function useRemainingTime(tx: MergedTransaction) {
     if (isTransferConfirmed(tx)) {
       setIsConfirmed(true)
     } else {
-      setRemainingTime(canBeClaimedDate.fromNow().toString())
+      setRemainingTime(canBeClaimedDate.fromNow(true).toString())
```

Comment:
We use `fromNow()` in other places (eg. txtablerow.tsx), whereas here, we use `fromNow(true)`. Are we sure it's supposed to be different in all these cases?

Reply:
This flag removes postfix if set to true. E.g.:
```
fromNow() => a year ago
fromNow(true) => a year
```

We don't remove it in the tx history row because we want to see the relative time, such as `a minute ago`.

We remove it in cctpState.ts because it's used in the the action column where `Time left:` is shown. So we want to display there `Time left: 10 minutes`, instead of `Time left: in 10 minutes`. That why the flags are different.